### PR TITLE
variants/msi_z690/building.md: Add building manual doc

### PR DESCRIPTION
**Pull request by IgorBagnucki**
Signed-off-by: Igor Bagnucki <igor.bagnucki@3mdeb.com>

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -81,6 +81,7 @@ nav:
           - 'Installation manual': variants/protectli_vp4620/installation-manual.md
           - 'Hardware Configuration Matrix': variants/protectli_vp4620/hardware-matrix.md
       - 'MSI PRO Z690-A WIFI DDR4':
+          - 'Building manual': variants/msi_z690/building.md
```
Comment (miczyg1) [inline mkdocs.yml:84]:
```suggestion
          - 'Building manual': variants/msi_z690/building-manual.md
```

I said the URL must be https://docs.dasharo.com/variants/msi_z690/building-manual/
Please change the file name accordingly

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: docs/variants/msi_z690/building.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# coreboot building
+
+## Intro
+
+This documents describes the procedure for compiling coreboot for MSI PRO Z690-A WIFI DDR4.
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/variants/msi_z690/building.md]: Fixed

File: docs/variants/msi_z690/building.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+Download and install UEFIExtract
+
+```
+sudo apt install wget
+wget https://github.com/LongSoft/UEFITool/releases/download/A59/UEFIExtract_NE_A59_linux_x86_64.zip
+unzip UEFIExtract_NE_A59_linux_x86_64.zip
+cp ./UEFIExtract /usr/bin
+```
+
+Obtain coreboot source code for MSI PRO Z690-A WIFI DDR4:
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/variants/msi_z690/building.md]: Fixed

File: docs/variants/msi_z690/building.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+```bash
+git clone https://github.com/Dasharo/coreboot.git -b msi_ms7d25/rel_v0.1.0
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/variants/msi_z690/building.md]: I see. Fixed that, but living the example of `rel_v0.1.0` tag later in the procedure, as this tag exists.

File: docs/variants/msi_z690/building.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+Navigate to the source code directory and checkout to the desired revision:
+
+> Replace the REVISION with one of the:
+> - `msi_ms7d25/release` for the latest released version
+> - `msi_ms7d25/rel_vVERSION` (e.g. `rel_v0.1.0`) for the given release
+
+```bash
+$ cd coreboot
+$ git checkout REVISION
+$ git submodule update --init --recursive --checkout
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/variants/msi_z690/building.md]: Removed this line

File: docs/variants/msi_z690/building.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+```bash
+$ ./build.sh build
+```
+The resulting coreboot image will be placed in
+`artifacts/dasharo_clevo_nv41mz_VERSION.rom`.
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/variants/msi_z690/building.md]: Overlooked that one. Fixed now

File: mkdocs.yml (hunk 1 above)
Comment (IgorBagnucki) [reply mkdocs.yml]: Fixed

File: docs/variants/msi_z690/building.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+This documents describes the procedure for compiling coreboot for MSI PRO Z690-A WIFI DDR4.
+
+## Requirements
+
+- Docker
+  + follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+  + follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+- Git
+
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/variants/msi_z690/building.md]: I have added the missing requrements

File: docs/variants/msi_z690/building.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+> - `msi_ms7d25/rel_vVERSION` (e.g. `rel_v0.1.0`) for the given release
+
+```bash
+$ cd coreboot
+$ git checkout REVISION
+$ git submodule update --init --recursive --checkout
+```
+
+```bash
+$ ./build.sh build
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/variants/msi_z690/building.md]: Fixed

File: docs/variants/msi_z690/building.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
+# coreboot building
+
+## Intro
+
+This documents describes the procedure for compiling coreboot for MSI PRO Z690-A WIFI DDR4.
+
+## Requirements
+
+- Docker
+  + follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
```
[the remark this answers is not in the document]
Comment (IgorBagnucki) [reply docs/variants/msi_z690/building.md]: Fixed

**Comment by macpijan** [comment]
Already included in this branch: https://github.com/Dasharo/docs/pull/145/commits

**Pull request closed without merging**